charts/argo-cd/templates/NOTES.txt

```diff
@@ -1,13 +0,0 @@
-In order to access the server UI you have the following options:
```

Comment:
I think the NOTES.txt do provide valuable information after you install the chart, for example showing you the generated admin password for the ui.

Reply:
Yes, thanks for catching that.  added back and updated: https://github.com/argoproj/argo-helm/pull/129/commits/79a683f8c61210340727332e2ec773557395fa95